montecarlo/SR0_parameters.ini

```diff
@@ -0,0 +1,308 @@
+[pax]
+parent_configuration = "pmt_afterpulsing"
+look_for_config_in_runs_db = False
+
+
+[WaveformSimulator]
+
+# S2 electron drift and extraction
+diffusion_constant_liquid =           22.8 * cm**2 / s   # Cathode at -12 kV, xenon:xenon1t:aalbers:drift_and_diffusion
+elr_gas_gap_length =                  2.66 * mm          # Average of level meters in run 3410
+s2_secondary_sc_gain =                23                 # photons/SE, xenon:xenon1t:aalbers:single_electron_gain_stability
+# Distance from anode where field becomes wire-like (~1/r) rather than uniform:
+anode_field_domination_distance =     0.372 * mm         # xenon:xenon1t:aalbers:single_electron_waveform_shape
+triplet_lifetime_gas      =           112.6               # +- 9.6 ns. xenon:xenon1t:aalbers:single_electron_waveform_shape
+gas_gap_warping_map =                 'gas_gap_warping_map_Oct16.pkl'   # Fit to single electrons
+
+# Used for s1 time structure calculations (currently not used; actually using empirical model):
+drift_field =                         124 * V /cm        
+
+# Light distribution
+s1_light_yield_map =                  's1_xyz_XENON1T_kr83m-nov_pax-642_fdc-AdCorrTPF.json'
+s2_light_yield_map =                  's2_xy_XENON1T_24Feb2017.json'
+s2_fitted_patterns_file =             'XENON1T_s2_xy_fitted_patterns_top_v0.3.0.json.gz'
+s2_mean_area_fraction_top =           0.680              # See XeAnalysisScripts/mapmaking
+
+# Various data-extracted corrections
+rz_position_distortion_map =          'fdc-AdCorrTPF.json.gz'
+
+
+[NeuralNet.PosRecNeuralNet]
+neural_net_file = 'NN_XENON1T_v5_n15.npz'
+
+[MC]
+mc_generated_data = True
+mc_sr_parameters = "sr0"
```

Comment:
This still needed?

Reply:
The mc_generated_data tag is used in both runs.py and corrections.py. The mc_sr_parameters tag is unused, but I would reccomend adding it in any case. It provides a simple tag to check what was done at a later stage,